ecosystem/sep-0006.md

```diff
@@ -276,32 +276,82 @@ Example:
 
 Response code: `403 Forbidden`
 
-An anchor that requires the user to fill out information on a webpage hosted by the anchor should use this response. A wallet that receives this response should open a popup browser window to the specified URL.
+An anchor that requires the user to fill out information on a webpage hosted by the anchor should use this response. This can happen in situations where the anchor needs KYC information about a user, or when the anchor needs the user to perform a custom step for each transaction like entering an SMS code to confirm a withdrawal or selecting a bank account. A wallet that receives this response should open a popup browser window to the specified URL. Next, the wallet must do one of two things:
```

Comment:
Should we mention somewhere here that these popups will likely run on mobile apps, so they need to be functional on major mobile browsers?

Reply:
Great call!